src/applications/virtual-agent/components/webchat/helpers/startConvoAndTrackUtterancesHelpers.js

```diff
@@ -104,9 +104,9 @@ export const processIncomingActivity = ({ action, dispatch }) => () => {
   const eventName = action?.payload?.activity?.name ?? '';
 
   // use event name for rxSkill
-  const rxSkillWasTriggered = eventName === 'RX_Skill_Entry';
+  const rxSkillWasTriggered = eventName === 'Skill_Entry';
```

Comment:
We should confirm that the Skill Entered is RX_Skill. We would need to check for the value as well. @nichelous-herndon

Reply:
good idea. I'll add the changes.

Reply:
@ksk385 Added the changes.